oslabs-beta/Kubiquity
# Abstract barchart

WHAT DOES THIS PR DO? 
1. Move BarChart into its own component. 

HOW TO TEST THIS PR:
1. `gh pr checkout 36 && npm install && npm run build && npm start`
2. Verify that the BarChart looks the same. 

ADDITIONAL NOTES: 
1. We are abstracting the specific MemoryBarChart component into a general BarChart component so that it can be used by multiple components.
